SimG4CMS/Calo/interface/CaloHitID.h

```diff
@@ -25,6 +25,11 @@ class CaloHitID {
   void setID(uint32_t unitID, double timeSlice, int trackID, uint16_t depth = 0);
   void reset();
 
+  void overwriteTrackID(int trackID) { theTrackID = trackID; }
```

Comment:
setters should be named `setX` (http://cms-sw.github.io/cms_coding_rules.html 2.10)

Reply:
Okay, changed it to `setTrackID`. I purposefully went with `overwrite` here because for the normal CMSSW workflow this property is assumed to be set initially and not changed after that (it's just fine fineCalo that you need to do this).